webapp-client/src/component/ContentPane.js

```diff
@@ -0,0 +1,67 @@
+import Category from './Category';
+import Item from './Item';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export default class ContentPane extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      displayCategory: <Category categoryName="All" />,
```

Comment:
maybe we shouldn't store a component in the state?

Reply:
I totally agree. I will take it out, but I initially didn't touch it because it is a holdover from the branch I came from. I am not sure that category switching is working in this PR